### DISCUSSION TITLE
Does anyone know if Tedious/TDS works with Oracle?

### DISCUSSION THREAD

**Discussion by lovetodream** [tediousjs/tedious#1512] [General]
I am wondering if anyone has experience using Tedious or TDS in general with an Oracle Database. I've found some (outdated) information online, indicating that Oracle might support TDS, but nothing useful.

**Comment by arthurschreiber** [comment]
I don't think that's possible. Are you sure you're not mixing this up with "Tax Deducted at Source" (TDS)?

**Reply by lovetodream** [reply]
Maybe, the stuff explicitly mentioning TDS wasn't first hand information. I've also found some info on ODBC directly from Oracle's docs and I assumed it uses TDS under the hood, which might be a wrong assumption

**Reply by lroal** [reply]
I recently added support for Oracle in the [RDB](https://github.com/alfateam/rdb) orm .
See the section named _Connecting_
You can use it for raw queries as well. It is using [oracledb](https://www.npmjs.com/package/oracledb) as the underlying driver.

```bash
$ npm install rdb oracledb
```

```js
import map from './map';
const db = map.oracle({
  user: 'sys',
  password: 'P@assword123',
  connectString: 'oracle/XE',
  privilege: 2
});
```
